composer.json

```diff
@@ -32,13 +32,14 @@
     "require": {
         "php": "^8.1|^8.2",
         "justbetter/statamic-glide-directive": "^2.1",
-        "rapidez/blade-directives": "^1.0",
+        "rapidez/blade-directives": "^0.6",
```

Comment:
Why downgrading?

Reply:
Because when testing in rapidez 2.x, the core requires 0.x, 1.x throws an installation error.

Reply:
It should work with both right?